addons/website_sale/controllers/variant.py

```diff
@@ -15,7 +15,7 @@ def get_combination_info_website(
         product_template = request.env['product.template'].browse(
             product_template_id and int(product_template_id))
 
-        combination_info = product_template._get_combination_info(
+        combination_info = product_template.with_context(display_default_code=False)._get_combination_info(
```

Comment:
Since `_get_combination_info` is only used in `website_sale` now, can't we fix it directly in the method ?

```python
display_name = product_or_template.with_context(display_default_code=False).display_name
```

When we can, we should aim to target as precisely as possible the context updates (it makes it easier to understand, and ensures that it only impacts the places we want).

Reply:
Done! :)